File: CS/LayoutGroupEx.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using DevExpress.Xpf.Docking;

namespace CustomLayoutGroup.Common {
    public class LayoutGroupEx : LayoutGroup {
        protected override GroupBorderStyle CoerceGroupBorderStyle(GroupBorderStyle value) {
            return value;
        }
    }
}

File: VB/LayoutGroupEx.vb
Imports System
Imports System.Collections.Generic
Imports System.Linq
Imports System.Text
Imports DevExpress.Xpf.Docking

Namespace CustomLayoutGroup.Common
    Public Class LayoutGroupEx
        Inherits LayoutGroup

        Protected Overrides Function CoerceGroupBorderStyle(ByVal value As GroupBorderStyle) As GroupBorderStyle
            Return value
        End Function
    End Class
End Namespace
